### PR TITLE
Replace compose with quick-start standalone OpenWhisk.

### PR DESCRIPTION
**Pull request by rabbah**
We've discussed in the past using the standalone controller as the new quick start. This morning I was trying to help someone in the community slack and I had trouble bringing up the compose version of the quick start. Plus it took many minutes to pull all the docker images. By contrast standalone is much lighter weight. 

This is an attempt to replace compose with standalone controller in the welcome docs.
cc @chetanmeh @sciabarra I know we can do even better with the docker self container version. If you can point me to those steps, I can include them.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [x] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [x] I added tests to cover my changes.
- [x] My changes require further changes to the documentation.
- [x] I updated the documentation where necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,46 +25,62 @@
...
 
+### Quick Start
 
-* [Quick Start](#quick-start) (Docker-Compose)
-* [Native development](#native-development) (Mac and Ubuntu)
-* [Kubernetes](#kubernetes-setup)
-* [Learn concepts and commands](#learn-concepts-and-commands)
-* [Issues](#issues)
-* [Slack](#slack)
+The easiest way to start using OpenWhisk is to install the "Standalone Controller". This is a full-featured OpenWhisk stack running as a Java
```
Comment (chetanmeh) [inline README.md:44]: Should we call it `Standalone Controller` or `Standalone OpenWhisk`? Referring to it as controller may confuse users to assume a limited feature set

File: README.md [hunk 2]
```diff
@@ -25,46 +25,62 @@
...
+process for convenience. Serverless functions run within Docker containers.
+Please refer to [Docker's installation instructions](https://docs.docker.com/install/) to configure your system to use Docker.
 
-### Quick Start
-The easiest way to start using OpenWhisk is to get Docker installed on Mac, Windows or Linux. The [Docker website](https://docs.docker.com/install/) has detailed instructions on getting the tools installed. This does not give you a production deployment but gives you enough of the pieces to start writing functions and seeing them run.
+```
+git clone https://github.com/apache/openwhisk.git
```
Comment (chetanmeh) [inline README.md:49]: Currently I think build may fail if node is not installed. Should we highlight that node and java are needed. Also not sure if build works fine on Windows

Best would have been to have users download a prebuilt jar but for that I think we need to wait for the core repo release.

File: README.md [hunk 3]
```diff
@@ -25,46 +25,62 @@
...
 
-For more detailed instructions or if you encounter problems see the [OpenWhisk-dev tools](https://github.com/apache/openwhisk-devtools/blob/master/docker-compose/README.md) project.
+- The Standalone Controller can be configured to deploy additional capabilities when that is desirable.
+Additional resources are available [here](./core/standalone/README.md).
 
-### Kubernetes Setup
+### Deploy to Kubernetes
 
-Another path to quickly starting to use OpenWhisk is to install it on a Kubernetes cluster.  On a Mac, you can use the Kubernetes support built into Docker 18.06 (or higher). You can also deploy OpenWhisk on Minikube, on a managed Kubernetes cluster provisioned from a public cloud provider, or on a Kubernetes cluster you manage yourself. To get started,
+Another path to quickly starting to use OpenWhisk is to install it on a Kubernetes cluster.
```
Comment (chetanmeh) [inline README.md:72]:
```
- Another path to quickly starting to use OpenWhisk
+ Another path to quickly start using OpenWhisk
```

File: README.md (hunk 1 above)
Comment (rabbah) [reply README.md:44]: Excellent point!

File: README.md (hunk 2 above)
Comment (rabbah) [reply README.md:49]: I will add Node and Java are also required. Agree on the jar, we need to push toward the release.

**Comment by rabbah** [comment]
Thanks Dave, Dominic and Chetan. Will incorporate the feedback today.

File: README.md [hunk 4]
```diff
@@ -25,46 +25,68 @@
...
+Additional resources are available [here](./core/standalone/README.md).
+
+### Deploy to Kubernetes
 
-### Kubernetes Setup
+OpenWhisk can also be installed on a Kubernetes cluster. You can use
+a managed Kubernetes cluster provisioned from a public cloud provider
+(e.g., AKS, EKS, IKS, GKE), or cluster you manage yourself.
+Additionally for local development, OpenWhisk is compatible with Minikube,
+and Kubernetes for Mac using the support built into Docker 18.06 (or higher).
```
Comment (rabbah) [inline README.md:79]: @dgrove-oss can you take another look at this paragraph?

Comment (dgrove-oss) [reply README.md:80]: Looks good.  Agree it makes more sense to lead with a managed cluster; that's the easy path for many users.

**Comment by rabbah** [comment]
Thanks for all the feedback. Will merge when Travis approves.

File: README.md (hunk 4 above)
Comment (rabbah) [reply README.md:78]: thanks.

File: README.md [hunk 5]
```diff
@@ -25,46 +25,67 @@
...
-For more detailed instructions or if you encounter problems see the [OpenWhisk-dev tools](https://github.com/apache/openwhisk-devtools/blob/master/docker-compose/README.md) project.
+- Standalone OpenWhisk can be configured to deploy additional capabilities when that is desirable.
+Additional resources are available [here](./core/standalone/README.md).
+
+### Deploy to Kubernetes
 
-### Kubernetes Setup
+OpenWhisk can also be installed on a Kubernetes cluster. You can use
+a managed Kubernetes cluster provisioned from a public cloud provider
+(e.g., AKS, EKS, IKS, GKE), or cluster you manage yourself.
```
Comment (rabbah) [inline README.md:76]:
```suggestion
(e.g., AKS, EKS, IKS, GKE), or a cluster you manage yourself.
```

**Pull request merged**